x-pack/plugins/observability_solution/apm/common/service_health_status.ts

```diff
@@ -34,29 +35,32 @@ export function getServiceHealthStatus({ severity }: { severity: ML_ANOMALY_SEVE
   }
 }
 
-export function getServiceHealthStatusColor(theme: EuiTheme, status: ServiceHealthStatus) {
+export function getServiceHealthStatusColor(theme: EuiThemeComputed, status: ServiceHealthStatus) {
   switch (status) {
     case ServiceHealthStatus.healthy:
-      return theme.eui.euiColorVis0;
+      return theme.colors.vis.euiColorVis0;
```

Comment:
Fyi, vis colors changed. So especially for severity colors like this it means using the same mapping will change colors in Amsterdam. This might need alignment with a designer (not sure which is responsible here?), if that's expected or not.

See this [related conversation](https://github.com/elastic/kibana/pull/204352#discussion_r1886896464) about this topic.

Reply:
@mgadewoll, thanks for the heads up! @maciejforcone is our designer, but I don't think we can get his feedback soon enough for the PR due to the holiday period.
 
However, I chose the 1st option in the related conversation you mentioned. a.k.a using `euiColorVisSuccess0`, `euiColorVisWarning0`, and `euiColorVisDanger0`

@jasonrhodes please let me know if you think we need to do otherwise.

Reply:
Fyi, we're [introducing new severity colors](https://github.com/elastic/eui/pull/8247) that should cover those cases across Kibana. 
We can update to severity colors as follow-up.